packages/server/lib/modes/interactive-ct.js

```diff
@@ -1,13 +1,32 @@
 const serverCt = require('@packages/server-ct')
+const { getBrowsers } = require('../browsers/utils')
 
-const run = (options) => {
-  const { projectRoot } = options
+const browsersForCtInteractive = ['chrome', 'chromium', 'edge', 'electron', 'firefox']
 
-  options.browser = options.browser || 'chrome'
+const returnDefaultBrowser = (browsersByPriority, installedBrowsers) => {
+  const browserMap = installedBrowsers.reduce((acc, curr) => {
+    acc[curr.name] = true
 
-  return serverCt.start(projectRoot, options)
+    return acc
+  }, {})
```

Comment:
This should be a `Set` instead

Reply:
Is there an objective reason that is better?

Reply:
It's cleaner and more clear

Reply:
Oops I merged, I'll make this change when I'm in this area again (prob soon). Thanks!